examples/sampling/graphbolt/lightning/node_classification.py

```diff
@@ -169,34 +187,42 @@ def val_dataloader(self):
     parser.add_argument(
         "--num_gpus",
         type=int,
-        default=4,
-        help="number of GPUs used for computing (default: 4)",
+        default=1,
+        help="number of GPUs used for computing (default: 1)",
     )
     parser.add_argument(
         "--batch_size",
         type=int,
-        default=32,
-        help="input batch size for training (default: 32)",
+        default=1024,
+        help="input batch size for training (default: 1024)",
     )
     parser.add_argument(
         "--epochs",
         type=int,
-        default=10,
-        help="number of epochs to train (default: 100)",
+        default=40,
+        help="number of epochs to train (default: 40)",
     )
     parser.add_argument(
         "--num_workers",
         type=int,
         default=0,
         help="number of workers (default: 0)",
     )
+    parser.add_argument(
+        "--labor",
```

Comment:
@mfbalin thanks for improving the example. This example is to showcase how graphbolt can work with lightning, maybe not worth to complicate it with more unrelated options. For showcase labor sampler, it is better to do it in the naive node_classfication example, https://github.com/dmlc/dgl/pull/6351 (our new intern is playing with it, but will be merged soon).

Reply:
Removed the option and made the default training sampler `sample_layer_neighbor`.